src/main/java/Circle.java

```diff
@@ -0,0 +1,23 @@
+public class Circle extends Figure implements AreaCalculator {
```

Comment:
```suggestion
public class Circle extends Figure {
```
пусть класс Figure имплементирует все необходимые интерфейсы, тогда все его наследники будут так же их импелементировать по умолчанию